### ISSUE TITLE
[1.12.2] Seems to download a version of custom mob spawner that requires 1.10.2

### ISSUE THREAD

**Issue by JordanLong** [DrZhark/mocreatures#123]
Downloading Mo'Creatures from the twitch app seems to download automatically, an older version of custom mob spawner that requires 1.10.2 (v3.10.4) when the 1.12.2 version requires the 1.12.2 (v3.11.4) version of custom mob spawner.

**Comment by ghost** [comment]
Yeah, can confirm. No updates for Custom Mob Spawner on Curseforge since 2016, therefore we can't use it on Twitch.

**Comment by RivaHakushia** [comment]
I had pointed this out end of last year as well [#113](https://github.com/DrZhark/mocreatures/issues/113)

At current you have to Manually add in the correct version, which can not be uploaded to curse if you make modpacks because they will not allow it :/ makes it a pain for modpack makers

**Comment by kreezxil** [comment]
confirmed confirmed confirmed and this actually bad for dr. zhark because so many of us would be adding it to our server packs and aka client packs.

**Comment by billyty08** [comment]
You dont need custom mob spawning mod in 1.12 version but it is set as a dependency so you need to manually remove it

**Comment by kreezxil** [comment]
someone with access power needs to get on cursforge and delist the
dependency if possible.

**Comment by RivaHakushia** [comment]
Either way I have personally removed this from my pack due to the hassle it causes, as much as I loved the mod it's not worth the effort on updating files etc.

**Comment by kreezxil** [comment]
exactly, this hassle ensures that DrZhark when he looks at CF demographics
wonders why nobody is interested in his mod. It's because us Pack Devs
download it, encounter the problem, report the issue, remove the mod and
move on. If this dependency issue were fixed, we'd download the mod and
leave it in place because then there would be no problem and he would see a
ton of downloads because lets face it's the best more creatures mod out. IE
being installed in pack that is popular like World of Dragons or something
similar would net at least an additional 4k downloads a week per equally
popular pack that has it.
